rollup-configs/integrationBuildScript.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-console */
```

Comment:
Let's move this file to a new directory (may be `scripts`) at the root level?

Reply:
sure will do this